src/app/header/header.component.html

```diff
@@ -27,13 +29,13 @@
                     </li>
                 </ul>
             </li>
-            <li *ngFor="let m of context.current.type.menus" [class.active]="m.active">
+            <li *ngFor="let m of context.current.type.menus"  [class.active]="m.active" [class.with-no-children]="(m.menus||[]).length===0" >
```

Comment:
the extra white space above scrollbar is there a a temp work around until all components have been tweaked to have a top padding of xpx. I just inserted a bottom padding in the header to make component content not lie right up agains header.